devbin/rotate_keys.py

```diff
@@ -70,7 +82,17 @@ def __init__(self, key_json):
         self.created_readable = key_json['validAfterTime']
         self.expiration_readable = key_json['validBeforeTime']
         self.created = isoparse(self.created_readable)
-        self.expiration = isoparse(self.expiration_readable)
+        assert key_json['keyType'] in ('USER_MANAGED', 'SYSTEM_MANAGED')
+        self.user_managed = key_json['keyType'] == 'USER_MANAGED'
+
+    def expired(self) -> bool:
+        return self.user_managed and self.older_than(90)
```

Comment:
That's fine. I originally had `self.user_managed` because I didn't want a system-managed key to show up as red, but looks like that was before I made system-managed keys greyed-out. Now looking through I don't call `expired` on any system-managed keys so shouldn't change anything.